modules/serverless-rn-1-8-0.adoc

```diff
@@ -9,14 +9,73 @@
 [id="new-features-1-8-0_{context}"]
 == New features
 
-* {ServerlessProductName} now uses Knative Serving 0.x
-* {ServerlessProductName} now uses Knative Serving Operator 0.x
-* {ServerlessProductName} now uses Knative `kn` CLI 0.x
-* {ServerlessProductName} uses Knative Eventing 0.x
-* {ServerlessProductName} now uses Knative Eventing Operator 0.x
+* {ServerlessProductName} now uses Knative Serving 0.14.1.
+* {ServerlessProductName} now uses Knative Serving Operator 0.14.0.
+* {ServerlessProductName} now uses Knative `kn` CLI 0.14.0.
+* {ServerlessProductName} uses Knative Eventing 0.14.2.
+* {ServerlessProductName} now uses Knative Eventing Operator 0.14.0.
+* {ServerlessProductName} now uses Kourier 0.14.1.
 
-[id="fixed-issues-1-8-0_{context}"]
-== Fixed issues
+// [id="fixed-issues-1-8-0_{context}"]
+// == Fixed issues
 
 [id="known-issues-1-8-0_{context}"]
 == Known issues
+* Knative Serving has a fixed, minimum CPU request of 25m for `queue-proxy`.
+If your cluster has any value set that conflicts with this, for example, if you have set a minimum CPU request for `defaultRequest` of more than `25m`, the Knative Service will fail to deploy.
+As a work-around, you can configure `resourcePercentage` individually for your Knative Services.
++
+.Example resourcePercentage configuration
++
+[source.yml]
+----
+spec:
+  template:
+    metadata:
+      annotations:
+        queue.sidecar.serving.knative.dev/resourcePercentage: "10"
+----
++
+`queue.sidecar.serving.knative.dev/resourcePercentage` is the percentage of user container resources to be used for `queue-proxy`. This can be between a range of 0.1 - 100.
+
+* On OpenShift 4.5 RC-6, deploying a Knative Service with traffic distribution will show an invalid URL for the general service address in the *Developer Perspective* of the web console.
+
+* If you use a PingSource with {ServerlessProductName}, after you uninstall and delete all other {ServerlessProductName} components, the `pingsource-jobrunner` Deployment is not deleted.
+
+* If you delete a sink before you delete the SinkBinding connected to it this may cause the SinkBinding deletion to hang.
++
+As a work-around, you can edit the SinkBinding and remove the finalizer that causes the hanging:
++
+
+[source,yml]
+----
+  finalizers:
+   - sinkbindings.sources.knative.dev
+----
+
+* The SinkBinding behavior has changed in {ServerlessProductName} 1.8.0, which breaks backwards compatibility.
++
+To use a SinkBinding, cluster administrators must now label the namespace and resources bound by a SinkBinding with `bindings.knative.dev/include:"true"`.
```

Comment:
Maybe more readable would be: "the namespace and resources mentioned in SinkBinding"

Reply:
technically, only the namespace must be labeled by an administrator ... the user should be able to label the resource by themselves.